tests/integration/targets/ec2_asg/roles/ec2_asg/tasks/create_update_delete.yml

```diff
@@ -511,10 +511,36 @@
 
     - assert:
         that:
-          - "output.mixed_instances_policy_full['launch_template']['overrides'][0]['instance_type'] == 't3.micro'"
-          - "output.mixed_instances_policy_full['launch_template']['overrides'][1]['instance_type'] == 't2.nano'"
-          - "output.mixed_instances_policy_full['instances_distribution']['on_demand_percentage_above_base_capacity'] == 0"
-          - "output.mixed_instances_policy_full['instances_distribution']['spot_allocation_strategy'] == 'capacity-optimized'"
+          - "output.mixed_instances_policy['launch_template']['overrides'][0]['instance_type'] == 't3.micro'"
+          - "output.mixed_instances_policy['launch_template']['overrides'][1]['instance_type'] == 't3a.micro'"
+          - "output.mixed_instances_policy['instances_distribution']['on_demand_percentage_above_base_capacity'] == 0"
+          - "output.mixed_instances_policy['instances_distribution']['spot_allocation_strategy'] == 'capacity-optimized'"
+
+    - name: update autoscaling group with mixed-instances policy with instances_requirements
+      ec2_asg:
+        name: "{{ resource_prefix }}-asg"
+        launch_template:
+          launch_template_name: "{{ resource_prefix }}-lt"
+        desired_capacity: 1
+        min_size: 1
+        max_size: 1
+        vpc_zone_identifier: "{{ testing_subnet.subnet.id }}"
+        state: present
+        mixed_instances_policy:
+          instance_requirements:
```

Comment:
Also seems that as [InstanceRequirements](https://github.com/ansible-collections/community.aws/pull/831/files#diff-629ed6761ca29636823559acc0c8f4aaa6c405c299e596ac5cddacc49447f569R952) was added in [1.19.5 of boto3 ](https://boto3.amazonaws.com/v1/documentation/api/1.19.5/reference/services/autoscaling.html#AutoScaling.Client.create_auto_scaling_group), could you also add boto3 version used by integration tests for this module?

so it would look something like these:

https://github.com/ansible-collections/community.aws/blob/main/tests/integration/targets/ec2_asg_scheduled_action/meta/main.yml

https://github.com/ansible-collections/community.aws/blob/main/tests/integration/targets/ec2_asg_scheduled_action/tasks/main.yml#L15